pages/api/service/configure_database.js

```diff
@@ -0,0 +1,15 @@
+// Load the AWS SDK for Node.js
```

Comment:
this file should probably not be in the` /pages` directory. Maybe put the `/service` folder in the main directory